src/labels/CustomLabels.labels

```diff
@@ -6076,6 +6368,67 @@ Selecting Don't Auto Schedule Default NPSP Jobs tells NPSP not to check for or r
         <shortDescription>stgHelpAccountModel</shortDescription>
         <value>The Account model used by the Nonprofit Success Pack. This option controls what happens when you create a Contact with a blank Account field value. WARNING: You should not use the Account Model record type you select as the default Account record type for any Profile. Doing so can cause various data issues.</value>
     </labels>
+    <labels>
+        <fullName>stgHelpAdvancedMapping1</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>stgHelpAdvancedMapping1</shortDescription>
+        <value>When you enable Advanced Mapping, we convert your existing Help Text field mapping in the NPSP Data Import object to use Custom Metadata. Once enabled, you'll edit mappings and create new ones on this page.</value>
+    </labels>
+    <labels>
+        <fullName>stgHelpAdvancedMapping2</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>stgHelpAdvancedMapping2</shortDescription>
+        <value>You can disable Advanced Mapping and go back to Help Text mapping, but any changes you make with Advanced Mapping will be lost.</value>
+    </labels>
+    <labels>
+        <fullName>stgHelpAdvancedMapping3</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>stgHelpAdvancedMapping3</shortDescription>
+        <value>Find complete Advanced Mapping setup documentation &lt;a href=&quot;https://powerofus.force.com/&quot; target=&quot;_blank&quot;&gt;here&lt;/a&gt;.</value>
```

Comment:
@psadie I'm checking with other doc writers to make sure I have this URL figured out correctly. I can't really test it until I have it published or built locally and I'm not quite there yet. Please ping me when you get to this comment to make sure I have the URL built correctly. 
```suggestion
        <value>Find complete Advanced Mapping setup documentation &lt;a href=&quot;https://powerofus.force.com/s/article/NPSP-Map-Using-Advanced-Mapping&quot; target=&quot;_blank&quot;&gt;here&lt;/a&gt;.</value>
```